compiler/rustc_query_impl/src/plumbing.rs

```diff
@@ -498,12 +510,25 @@ macro_rules! define_queries {
                         mode
                     )
                 }
+
+                pub mod $name {
+                    use super::*;
+                    #[inline(never)]
+                    pub fn __rust_end_short_backtrace<'tcx>(
+                        tcx: TyCtxt<'tcx>,
+                        span: Span,
+                        key: query_keys::$name<'tcx>,
+                        mode: QueryMode,
+                    ) -> Option<Erase<query_values::$name<'tcx>>> {
+                        $name(tcx, span, key, mode)
```

Comment:
Could you inline the `$name` functions into this one. This is their single call point anyway.